src/gui/mzroll/tabledockwidget.cpp

```diff
@@ -323,6 +360,26 @@ void TableDockWidget::addRow(PeakGroup *group, QTreeWidgetItem *root) {
       item->setText(15, QString::number(group->changeFoldRatio, 'f', 2));
       item->setText(16, QString::number(group->changePValue, 'e', 4));
     }
+    item->setText(15,QString::number(group->avgPeakQuality,'f',2));
```

Comment:
@Ky5595 Column number 15 and 16 are being overwritten here. Please resolve this.

Reply:
@Ky5595 Please take care of this.